modules/administration-guide/examples/snip_using-devworkspace-warning.adoc

```diff
@@ -0,0 +1,10 @@
+[WARNING]
+.Technology preview feature
```

Comment:
Should you be using the official language, which is covered in 
openshift/openshift-docs/contributing_to_docs/doc_guidelines.adoc?
```
== Indicating Technology Preview features

To indicate that a feature is in Technology Preview, include the
`modules/technology-preview.adoc` file in the feature's assembly to keep the
supportability wording consistent across Technology Preview features and provide a value for the :FeatureName: variable before you include this module.

See
link:https://github.com/openshift/openshift-docs/pull/13878/files#diff-615ba1bf3b09d11a9c2604b775aa32f2[an
example] of how this is applied.

Reply:
I didn't know about that. We don't have this implemented yet here. I will take a look.